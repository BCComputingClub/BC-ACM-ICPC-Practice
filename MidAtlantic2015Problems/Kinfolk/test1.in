22 191 F
191 22 F
1 5 M
1 11 F
0 8 F
5 7 M
0 32767 F
5002 2500 M
5002 1249 F
5002 624 M
4998 624 F
16000 32002 M
16001 32002 F
16003 32002 M
16009 32002 F
22 23 F
22 23 M
23 22 F
23 22 M
45 49 F
45 49 M
49 45 F
49 45 M
22 33 F
22 33 M
33 22 F
33 22 M
45 71 F
45 71 M
71 45 F
71 45 M
92 148 F
92 148 M
148 92 F
148 92 M
22 44 F
22 44 M
44 22 F
44 22 M
22 45 F
22 45 M
45 22 F
45 22 M
45 93 F
45 93 M
93 45 F
93 45 M
92 192 F
92 192 M
192 92 F
192 92 M
22 55 F
22 55 M
55 22 F
55 22 M
45 115 F
45 115 M
115 45 F
115 45 M
22 65 F
22 65 M
65 22 F
65 22 M
45 135 F
45 135 M
135 45 F
135 45 M
92 276 F
92 276 M
276 92 F
276 92 M
22 75 F
22 75 M
75 22 F
75 22 M
45 155 F
45 155 M
155 45 F
155 45 M
92 316 F
92 316 M
316 92 F
316 92 M
22 85 F
22 85 M
85 22 F
85 22 M
92 356 F
92 356 M
356 92 F
356 92 M
185 721 F
185 721 M
721 185 F
721 185 M
372 1460 F
372 1460 M
1460 372 F
1460 372 M
745 2953 F
745 2953 M
2953 745 F
2953 745 M
1492 5972 F
1492 5972 M
5972 1492 F
5972 1492 M
2985 12073 F
2985 12073 M
12073 2985 F
12073 2985 M
22 95 F
22 95 M
95 22 F
95 22 M
45 195 F
45 195 M
195 45 F
195 45 M
22 105 F
22 105 M
105 22 F
105 22 M
45 215 F
45 215 M
215 45 F
215 45 M
22 115 F
22 115 M
115 22 F
115 22 M
45 235 F
45 235 M
235 45 F
235 45 M
22 125 F
22 125 M
125 22 F
125 22 M
45 255 F
45 255 M
255 45 F
255 45 M
92 516 F
92 516 M
516 92 F
516 92 M
22 134 F
22 134 M
134 22 F
134 22 M
45 273 F
45 273 M
273 45 F
273 45 M
92 552 F
92 552 M
552 92 F
552 92 M
22 143 F
22 143 M
143 22 F
143 22 M
45 291 F
45 291 M
291 45 F
291 45 M
92 588 F
92 588 M
588 92 F
588 92 M
22 152 F
22 152 M
152 22 F
152 22 M
45 309 F
45 309 M
309 45 F
309 45 M
92 624 F
92 624 M
624 92 F
624 92 M
22 161 F
22 161 M
161 22 F
161 22 M
45 327 F
45 327 M
327 45 F
327 45 M
92 660 F
92 660 M
660 92 F
660 92 M
185 1329 F
185 1329 M
1329 185 F
1329 185 M
22 170 F
22 170 M
170 22 F
170 22 M
45 345 F
45 345 M
345 45 F
345 45 M
92 696 F
92 696 M
696 92 F
696 92 M
185 1401 F
185 1401 M
1401 185 F
1401 185 M
22 179 F
22 179 M
179 22 F
179 22 M
45 363 F
45 363 M
363 45 F
363 45 M
92 732 F
92 732 M
732 92 F
732 92 M
185 1473 F
185 1473 M
1473 185 F
1473 185 M
372 2964 F
372 2964 M
2964 372 F
2964 372 M
745 5961 F
745 5961 M
5961 745 F
5961 745 M
1492 11988 F
1492 11988 M
11988 1492 F
11988 1492 M
2985 24105 F
2985 24105 M
24105 2985 F
24105 2985 M
22 188 F
22 188 M
188 22 F
188 22 M
45 381 F
45 381 M
381 45 F
381 45 M
92 768 F
92 768 M
768 92 F
768 92 M
22 197 F
22 197 M
197 22 F
197 22 M
45 399 F
45 399 M
399 45 F
399 45 M
22 206 F
22 206 M
206 22 F
206 22 M
45 417 F
45 417 M
417 45 F
417 45 M
22 215 F
22 215 M
215 22 F
215 22 M
45 435 F
45 435 M
435 45 F
435 45 M
22 224 F
22 224 M
224 22 F
224 22 M
45 453 F
45 453 M
453 45 F
453 45 M
22 233 F
22 233 M
233 22 F
233 22 M
45 471 F
45 471 M
471 45 F
471 45 M
22 242 F
22 242 M
242 22 F
242 22 M
45 489 F
45 489 M
489 45 F
489 45 M
22 251 F
22 251 M
251 22 F
251 22 M
45 507 F
45 507 M
507 45 F
507 45 M
22 355 F
22 355 M
355 22 F
355 22 M
45 715 F
45 715 M
715 45 F
715 45 M
22 363 F
22 363 M
363 22 F
363 22 M
45 731 F
45 731 M
731 45 F
731 45 M
22 371 F
22 371 M
371 22 F
371 22 M
185 3009 F
185 3009 M
3009 185 F
3009 185 M
22 379 F
22 379 M
379 22 F
379 22 M
45 763 F
45 763 M
763 45 F
763 45 M
92 1532 F
92 1532 M
1532 92 F
1532 92 M
-1 -1 F
