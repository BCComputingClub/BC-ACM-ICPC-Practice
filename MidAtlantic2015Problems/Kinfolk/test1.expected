3rd cousin thrice removed
3rd cousin thrice removed
nephew
grandniece
great-grandchild
1st cousin once removed
kin
parent
grandparent
great-grandparent
great-grandaunt
child
1st cousin once removed
2nd cousin once removed
3rd cousin once removed
3rd cousin
3rd cousin
3rd cousin
3rd cousin
kin
kin
kin
kin
2nd cousin once removed
2nd cousin once removed
2nd cousin once removed
2nd cousin once removed
3rd cousin once removed
3rd cousin once removed
3rd cousin once removed
3rd cousin once removed
kin
kin
kin
kin
niece
nephew
aunt
uncle
child
child
parent
parent
niece
nephew
aunt
uncle
kin
kin
kin
kin
3rd cousin once removed
3rd cousin once removed
3rd cousin once removed
3rd cousin once removed
kin
kin
kin
kin
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
1st cousin twice removed
1st cousin twice removed
1st cousin twice removed
1st cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
2nd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
grandchild
grandchild
grandparent
grandparent
kin
kin
kin
kin
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
kin
kin
kin
kin
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin twice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
great-grandniece
great-grandnephew
great-grandaunt
great-granduncle
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
2nd cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
1st cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
great-grandchild
great-grandchild
great-grandparent
great-grandparent
great-grandniece
great-grandnephew
great-grandaunt
great-granduncle
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
3rd cousin thrice removed
kin
kin
kin
kin
great-great-grandniece
great-great-grandnephew
great-great-grandaunt
great-great-granduncle
kin
kin
kin
kin
great-great-grandniece
great-great-grandnephew
great-great-grandaunt
great-great-granduncle
kin
kin
kin
kin
great-great-grandchild
great-great-grandchild
great-great-grandparent
great-great-grandparent
kin
kin
kin
kin
great-great-grandchild
great-great-grandchild
great-great-grandparent
great-great-grandparent
great-great-grandniece
great-great-grandnephew
great-great-grandaunt
great-great-granduncle
kin
kin
kin
kin
